src/RFC-0314-VNCSelection.md

```diff
@@ -0,0 +1,100 @@
+technological merits of the potential system outlined herein.
+
+## Goals
+The goal of this RFC is to describe the process for allocating Validator Nodes (VNs) to Validator Node Committees (VNCs)
+
+## Intro
+Validator nodes will have to group themselves into Validator Node Committees for transactions/shard processing. Committees will be formed for each new transaction/shard processing. This committee needs to be determined pseudorandomly, and we use the VN_Key as this changes periodically and is pseudorandom. Each VN will be allocated a unique and individual shard space to serve in as a VNC member. 
+
+## Committee Creation
+Because we have the base layer where each VN needs to publish a registration transaction, we can get base_node and miners to keep track of all active VNs. We represent all VNs in a (balanced) Merkle tree with the VN_keys as leaves. We declare a constant `COMMITTEE_SIZE` which can be changed in the consensus constants.
+For the sake of simplicity, `COMMITTEE_SIZE` MUST be an even positive integer.
+
+As per the Cerberus algorithm, validator nodes are responsible for managing _sub-states_, rather than contract semantics. 
+A given instruction may involve dozens of sub-states, meaning that there are potentially dozens of non-overlapping committees that are required to reach a braided consensus.
+Each committee is determined independently. For each sub-state, a committee is formed by taking the first  `COMMITTEE_SIZE / 2` VN_keys to the left and `COMMITTEE_SIZE / 2` VN_Keys to the right of the sub-state's shard address in the merkle tree.
+
+This makes it very easy to determine what shard space a VN needs to serve, which states the VN needs to sync from peers, and who has them. Because the
+second layer has a delayed view of the network. VNs can also know beforehand and prep to ensure they are ready when new block heights appear. 
+
+An important edge case here that is implied but not listed, is that if the whole network is less or equal to the `COMMITTEE_SIZE`, then the whole network participates in the VNC.
+
+## Committee proofs
+We construct the balanced Merkle tree from all active VN registration transactions and prune away all inactive ones. Because the [base node]s have to keep track
+of the entire unspent UTXO set, it becomes easy for them to track and validate all active VN registration UTXOs. This means we can keep [base node]s 
+responsible for validating and constructing the Merkle tree. We commit this Merkle tree per block as a Merkle root inside the block's header as the `validator_node_set_root`. 
+
+This Merkle root in the header always lags by one block, meaning that the Merkle root is for the state of the VN's before the start of the block it's mined in. 
+When a VN registers, that new VN key will only appear inside the Merkle tree in the next block. The reason for this is that header_hash is used in the
+the calculation process of the VN_key. 
+
+### Option 2 for Committee proofs

```

Comment:
This section should be removed and merged with the above section, which needs more detail as @CjS77 suggests